template-for-readme.md

```diff
@@ -20,13 +20,13 @@
     * Other Version Control Information - (enter, if applicable)
 
 ## Tool Installation: Name of Tool
-(link for download, if applicable)
-(link for source code, if applicable)
-(link for installation instructions, if applicable)
-(link for documentation, if applicable)
-(link for user manual, if applicable)
-(Any intro needed for tool)
-
+(link for download, if applicable)  
+(link for source code, if applicable)  
+(link for installation instructions, if applicable)  
+(link for documentation, if applicable)  
+(link for user manual, if applicable)  
+(Any intro needed for tool)  
+  
```

Comment:
I believe these are probably nothing, but just wanted to make sure that these (non-)changes are not related to any changes you might have made to address line break problem. Or, are they?

Reply:
I simply added two spaces to make these all appear on different lines so that in the future it is easier to grab and replace with the real link text. 

(Has nothing to do with file-formatting, LF etc)